docs/integrations/asgi.md

```diff
@@ -27,19 +27,19 @@ The `GraphQL` app accepts two options at the moment:
 
 We allow to extend the base `GraphQL` app, by overriding the following methods:
 
-- `async get_context(self, request: Union[Request, WebSocket]) -> Any`
+- `async get_context(self, request: Union[Request, WebSocket], response: typing.Optional[Response] = None) -> Any`
 - `async get_root_value(self, request: Request) -> Any`
 - `async process_result(self, request: Request, result: ExecutionResult) -> GraphQLHTTPResponse`
 
 ## get_context
 
 `get_context` allows to provide a custom context object that can be used in your
 resolver. You can return anything here, by default we return a dictionary with
-the request.
+the request and the response.
 
-```python
+```python3
```

Comment:
`python3` is equivalent to `python`. No need for this change

Reply:
Also should change the new code block below to use `python` instead of `python3`